modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -627,7 +627,7 @@ class ApiClient(object):
             auth_setting = self.configuration.auth_settings().get(auth)
             if auth_setting:
                 if auth_setting['in'] == 'cookie':
-                    headers['Cookie'] = auth_setting['value']
+                    headers['Cookie'] = auth_setting['key'] + "=" + auth_setting['value']
```

Comment:
Another way is to set the `auth_setting['value']` directly to `auth_setting['key'] + "=" + auth_setting['value']`

Reply:
Sure, I could do that. Just to be sure: You mean directly before the line I changed, right? Because this should only be relevant for the cookie case afaik.
```
                if auth_setting['in'] == 'cookie':
                    auth_setting['value'] = auth_setting['key'] + "=" + auth_setting['value']
                    headers['Cookie'] = auth_setting['value']
```

I'm not sure which benefits this brings, but I can change it. Just wanted to make sure that this is the change you want.

Reply:
Sorry I was not clear. I mean when users of the SDK set the `value` of `auth_setting`, they will include `auth_setting['key']` directly.

In other words, there's no need to change any of the code.

Another way is to use customized templates (e.g. -t option via CLI) if you want to make it easier. for your SDK users.

Reply:
I'm afraid I still don't get it. AFAIk I can use -t to specify my own directory of templates, which is what I'm doing right now so that I can get a working API client for my Resteasy server.
The following part of the comment is under the assumption that you essentially want to leave the current behavior as is.
Sure, you can leave it as it is right now, but it does not change the fact that it is a miracle that it works at all for some server implementations. How is the server supposed to know which cookie has the value if the client does not send the name of the cookie?
Directly copying from wikipedia, a HTTP request should send a cookie like
```
GET /spec.html HTTP/1.1
Host: www.example.org
Cookie: theme=light
```
and not 
```
GET /spec.html HTTP/1.1
Host: www.example.org
Cookie: light
```
The latter is what the python api client is currently generating.

Other code generations (e.g. for Java) do this correctly.
https://github.com/OpenAPITools/openapi-generator/blob/37905e8bfac75aab0902f0ca0d50de2e070ab544/modules/openapi-generator/src/main/resources/Java/ApiClient.mustache#L763
The java apiclient mustache file uses the cookie api call to specify the cookie value _and the cookie name_. The http framework uses both to generate a HTTP header which looks like it should. As a cookie is a key-value pair, there really should be both in the Request's cookie header.

In contrast, Python's mustache file (as of now) only uses the cookie value and generates a non-standard compliant cookie header. Instead of a key-value pair, it only sends the value. The server may or may not handle this non-standard behavior in some way, but even if it works, it is just due to the implementation and interpretation of this non-standard header value and should not be relied upon. The implementation I'm using (JBoss RESTEasy), which is used by any people, does not accept such invalid cookie headers.

Reply:
@MarcMil thanks for the detailed explanation. I think I now understand what the problem is.

I agree your PR is more like bug fix. Can you please resolve the merge conflicts?

We'll merge your fix and later introduce an option for fallback if needed.